userguide/drawing_classifier/export-coreml.md

```diff
@@ -34,6 +34,18 @@ let model = try VNCoreMLModel(for: MySquareTriangleClassifier().model)
 let request = VNCoreMLRequest(model: model, completionHandler: { [weak self] request, error in
     self?.processClassifications(for: request, error: error)
 })
+let grayscaleImage = UIImage(named: "MyImage")
+        
+let handler = VNImageRequestHandler(cgImage: grayscaleImage.cgImage!, options: [:])
```

Comment:
I get the error: `Value of optional type 'UIImage?' must be unwrapped to refer to member 'cgImage' of wrapped base type 'UIImage'`